rapid7_insightvm/plugin.spec.yaml

```diff
@@ -1644,6 +1729,18 @@ types:
       description: The vulnerabilities that would be remediated
       type: '[]remediation_vulnerability'
       required: true
+  vulnerability_solution:
+    links:
+      description: Hypermedia links to corresponding or related resources
+      title: Links
+      type: '[]link'
```

Comment:
```suggestion
      type: '[]resources'
```
Do you want the resources type here?

Reply:
It was the other object that needed to be resources, but thanks for making me check. I had forgotten to change it from object to resources

Reply:
oh yeah, just linked to the wrong line when I did it 🤦‍♂️